docs/orbit_registration.md

```diff
@@ -0,0 +1,142 @@
+
+## Minikube and MetalK8s (and other Kubernetes-based deployments)
+
+### Step 1: go to the Kubernetes dashboard
+
+##### For minikube:
+```
+$ minikube dashboard
+```
+A new tab should open in your browser, taking you to the Kubernetes dashboard.
+
+##### For MetalK8s:
+```
+$ kubectl proxy
+```
+While the tunnel is up and running, access the dashboard at
+[http://localhost:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/](http://localhost:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/).
+
+### Step 2: finding the Zenko instance ID
+
+#### Option A: from the CloudServer logs
+
+From the dashboard, click on the cloudserver-front service, and then on one of
+the pods running for that service. On the top right menu, click "Logs", and then
+do a search of "instance ID" - you should find a line in the logs giving you a
+long string of random characters: this is your instance id, copy it. If you
+cannot find such a line in the logs, try option B.
+
+#### Option B: from the Metadata MongoDB cluster
+
+From the dashboard, click on the mongodb-replicaset service, and then on the
+pod that is mongodb-replicaset-**0**. This is important as the other pods are
+slaves to that master pod 0, so you will not be able to query the MongoDB
+database from them.
+
+Then hit "Exec", in the top right menu of the dashboard, and use the CLI to
+query MongoDB as follows
+
+``` shell
+$ mongo
+MongoDB shell version v3.4.15
+connecting to: mongodb://127.0.0.1:27017
+MongoDB server version: 3.4.15
+Welcome to the MongoDB shell.
+For interactive help, type "help".
+For more comprehensive documentation, see
+        http://docs.mongodb.org/
+Questions? Try the support group
+        http://groups.google.com/group/mongodb-user
+Server has startup warnings: 
+2018-05-25T15:44:08.084+0000 I STORAGE  [initandlisten] 
+2018-05-25T15:44:08.084+0000 I STORAGE  [initandlisten] ** WARNING: Using the XFS filesystem is strongly recommended with the WiredTiger storage engine
+2018-05-25T15:44:08.084+0000 I STORAGE  [initandlisten] **          See http://dochub.mongodb.org/core/prodnotes-filesystem
+2018-05-25T15:44:08.503+0000 I CONTROL  [initandlisten] 
+2018-05-25T15:44:08.503+0000 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
+2018-05-25T15:44:08.504+0000 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
+2018-05-25T15:44:08.504+0000 I CONTROL  [initandlisten] ** WARNING: You are running this process as the root user, which is not recommended.
+2018-05-25T15:44:08.504+0000 I CONTROL  [initandlisten] 
+
+rs0:PRIMARY> use metadata
+switched to db metadata
+
+rs0:PRIMARY> db.PENSIEVE.findOne({_id: "auth/zenko/remote-management-token"})
+{
+    "_id" : "auth/zenko/remote-management-token",
+    "value" : {
+        "instanceId" : "f1a846d6-b115-4f15-b0e6-5a9c17e7ae01",
+        "issueDate" : "2018-05-25T15:45:56Z",
```

Comment:
maybe something more like this? It takes up less screen space when reading and gets rid of unnecessary info for finding the ID =)
```
{
     "_id" : "auth/zenko/remote-management-token",
     "value" : {
         "instanceId" : "f1a846d6-b115-4f15-b0e6-5a9c17e7ae01",
         "issueDate" : "2018-05-25T15:45:56Z",
         ...
      }
}
```

Reply:
A dummy instance ID that can't be registered might be a good idea so people can't register the example ID.

Reply:
Yes, of course ;) You reviewed really fast, which is cool, so I hadn't crashed the instance from which this id came - it's done, registration should now be disabled

Shortened paragraph about mongo query

Reply:
Isn't that a bug then? I'll change the id, but might be worth opening an Orbit issue for "Destroyed stack ids are not forgotten", no?

Reply:
I'm not sure if it's intended to be this way but you're right it'd be worth opening an issue.